Config/FilterPlugin.ini

```diff
@@ -0,0 +1,8 @@
+[FilterPlugin]
```

Comment:
How come this file was added? Looks like everything is commented out?

Reply:
I don't even know what this is or where it came from :(

Reply:
Did you autogenerate / via the editor a plugin? I assume then this is an autogenerated config file.

Reply:
Can we remove this then if we don't want it?